prototype/Makefile

```diff
@@ -40,3 +44,7 @@ clean_all: clean_venv
 .PHONY: run
 run:
 	paster
+
+.PHONY: test
+test:	deps
+	bin/nosetests -x -s jenkinsmerger
```

Comment:
wow, I'd completely forgotten about the prototype bits and the jenkinsmerger stuff here. Is this even needed or is this just dead code?

/edit I guess it was the web based tool and it's split from the lander itself so it's still there.